.github/workflows/docker.yml

```
@@ -67,6 +74,7 @@ jobs:
             COMMIT_SHA=${{ github.sha }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
+          qemu-version: ${{ matrix.build.arch == 'amd64' && '7' || '' }}
```

actionlint (1.7.8)

input "qemu-version" is not defined in action "docker/build-push-action@v6". available inputs are "add-hosts", "allow", "annotations", "attests", "build-args", "build-contexts", "builder", "cache-from", "cache-to", "call", "cgroup-parent", "context", "file", "github-token", "labels", "load", "network", "no-cache", "no-cache-filters", "outputs", "platforms", "provenance", "pull", "push", "sbom", "secret-envs", "secret-files", "secrets", "shm-size", "ssh", "tags", "target", "ulimit"

(action)